framework/src/fvkernels/FVAnisotropicDiffusion.C

```diff
@@ -24,32 +24,41 @@ FVAnisotropicDiffusion::validParams()
       "coeff_interp_method",
       coeff_interp_method,
       "Switch that can select face interpolation method for diffusion coefficients.");
+  MooseEnum face_interp_method("average skewness-corrected", "average");
+  params.addParam<MooseEnum>(
+      "variable_interp_method",
+      face_interp_method,
+      "Switch that can select between face interpolation methods for the variable.");
```

Comment:
I think we should either move this to the fv relationship manager interface or error check the retrieval of `variable_interp_method` in that interface and give some meaningful error context like "you tried to set diffusion parameters from a non-diffusion class" or something